src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect/NetworkInterfaceNetworkSelect.tsx

```diff
@@ -110,8 +115,9 @@ const NetworkInterfaceNetworkSelect: FC<NetworkInterfaceNetworkSelectProps> = ({
       setSubmitDisabled(false);
     }
     // if networkName is empty, and pod network exists we can create a NIC with existing NAD if there is one
-    else if (loaded && !loadError) {
+    else if (loaded && !loadError && !selectedFirstOnLoad.current) {
```

Comment:
no need this else if in this effect, add return statements.

Reply:
This is necessary as if you clear the input and have no `networkName`, this effect will select automatically the first one. Initially it's helpful but after the initial set, it make impossible to clear the input.

Reply:
i mean you can remove the elseif you u add return for example after selectedFirstOnLoad.current = true; return;